docs/src/modules/components/BaseUIComponents.js

```diff
@@ -4,136 +4,136 @@ import CardMedia from '@mui/material/CardMedia';
-      href: '/base/react-portal/',
+      srcLight: '/static/base-ui/react-components/portal-light.png',
+      srcDark: '/static/base-ui/react-components/portal-dark.png',
+      href: '/base-ui/react-portal/',
     },
     {
       title: 'Select',
-      srcLight: '/static/images/base-ui-components/select-light.png',
-      srcDark: '/static/images/base-ui-components/select-dark.png',
-      href: '/base/react-select/',
+      srcLight: '/static/base-ui/react-components/select-light.png',
+      srcDark: '/static/base-ui/react-components/select-dark.png',
+      href: '/base-ui/react-select/',
     },
     {
       title: 'Slider',
-      srcLight: '/static/images/base-ui-components/slider-light.png',
-      srcDark: '/static/images/base-ui-components/slider-dark.png',
-      href: '/base/react-slider/',
+      srcLight: '/static/base-ui/react-components/slider-light.png',
+      srcDark: '/static/base-ui/react-components/slider-dark.png',
+      href: '/base-ui/react-slider/',
     },
     {
       title: 'Snackbar',
-      srcLight: '/static/images/base-ui-components/snackbar-light.png',
-      srcDark: '/static/images/base-ui-components/snackbar-dark.png',
-      href: '/base/react-snackbar/',
+      srcLight: '/static/base-ui/react-components/snackbar-light.png',
+      srcDark: '/static/base-ui/react-components/snackbar-dark.png',
+      href: '/base-ui/react-snackbar/',
     },
     {
       title: 'Switch',
-      srcLight: '/static/images/base-ui-components/switch-light.png',
-      srcDark: '/static/images/base-ui-components/switch-dark.png',
-      href: '/base/react-switch/',
+      srcLight: '/static/base-ui/react-components/switch-light.png',
+      srcDark: '/static/base-ui/react-components/switch-dark.png',
+      href: '/base-ui/react-switch/',
     },
     {
       title: 'Table Pagination',
-      srcLight: '/static/images/base-ui-components/pagination-light.png',
-      srcDark: '/static/images/base-ui-components/pagination-dark.png',
-      href: '/base/react-table-pagination/',
+      srcLight: '/static/base-ui/react-components/pagination-light.png',
+      srcDark: '/static/base-ui/react-components/pagination-dark.png',
+      href: '/base-ui/react-table-pagination/',
     },
     {
       title: 'Tabs',
-      srcLight: '/static/images/base-ui-components/tabs-light.png',
-      srcDark: '/static/images/base-ui-components/tabs-dark.png',
-      href: '/base/react-tabs/',
+      srcLight: '/static/base-ui/react-components/tabs-light.png',
+      srcDark: '/static/base-ui/react-components/tabs-dark.png',
+      href: '/base-ui/react-tabs/',
     },
     {
       title: 'Textarea Autosize',
-      srcLight: '/static/images/base-ui-components/textarea-light.png',
-      srcDark: '/static/images/base-ui-components/textarea-dark.png',
-      href: '/base/react-textarea-autosize/',
+      srcLight: '/static/base-ui/react-components/textarea-light.png',
+      srcDark: '/static/base-ui/react-components/textarea-dark.png',
```

Comment:
I think that we can reorganize the images to have a root folder per product, in this case /base-ui/